graph/index.html

```diff
@@ -65,5 +65,21 @@
 <script src="//cdn.rawgit.com/bassjobsen/Bootstrap-3-Typeahead/master/bootstrap3-typeahead.min.js"></script>
 <script src="main.js"></script>
 <script>main();</script>
+<!-- Piwik -->
+<script type="text/javascript">
+  var _paq = _paq || [];
+  _paq.push(["setDomains", ["*.openfisca.fr","*.legislation.openfisca.fr","*.ui.openfisca.fr","*.www.openfisca.fr"]]);
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//stats.data.gouv.fr/";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', '4']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+  })();
+</script>
+<noscript><p><img src="//stats.data.gouv.fr/piwik.php?idsite=4" style="border:0;" alt="" /></p></noscript>
```

Comment:
Please enforce `HTTPS` in images. Schema-less is deprecated, and we intend to only serve over HTTPS anyway.